### PR TITLE
Added the UserBadgeLg Component

### PR DESCRIPTION
**Pull request by goldyydev**
This PR
- Adds the UserBadgeLg Component
- Is dependent on #2033 for Storybook to work
- Related to #2022

### CODE REVIEW

**Review by BronzW — APPROVED** [review]
Looks good

File: kibbeh/src/ui/UserBadgeLg.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { SolidDogenitro } from "../icons";
+import Logo from "../icons/LogoIcon";
+
+export interface UserBadgeLgProps extends React.HTMLProps<HTMLDivElement> {
+  icon: "logo" | "dogeNitro" | "dogeStaff" | "dogeContributor";
+}
+
+// @todo the tag doesn't really glow like in figma right now
```
Comment (amitojsingh366) [inline kibbeh/src/ui/UserBadgeLg.tsx]: wdym by glow? i dont see any glow here https://www.figma.com/file/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=8277%3A990

Comment (goldyydev) [reply kibbeh/src/ui/UserBadgeLg.tsx]: Not this line again....

Comment (goldyydev) [inline kibbeh/src/ui/UserBadgeLg.tsx]:
```suggestion
```

Comment (goldyydev) [reply kibbeh/src/ui/UserBadgeLg.tsx]: Fixed

**Comment by goldyydev** [comment]
Merging this as this has been reviewed and follows the Figma design

**Pull request merged**